libraries/vdf/src/sortition.cpp

```diff
@@ -11,8 +11,7 @@ VdfSortition::VdfSortition(const SortitionParams& config, const vrf_sk_t& sk, co
 }
 
 bool VdfSortition::isOmitVdf(SortitionParams const& config) const {
-  return config.vrf.threshold_upper >= config.vrf.threshold_range &&
-         threshold_ <= config.vrf.threshold_upper - config.vrf.threshold_range;
+  return threshold_ <= config.vrf.threshold_upper * (100 - config.vrf.threshold_range) / 100;
```

Comment:
I don't get this.
threshold_upper is 45055, threshold_range is 50
45055 * (100 - 50) / 100 means 50% of threshold_upper
 Does this all threshold_ <= 1/2 * threshold_upper that is omit VDF? That's a lot

Reply:
Yes, I put it as default that 50% of the range is omit vdf. Before my change it was even bigger.